src/Symfony/Component/Validator/Constraints/ExpressionLanguageSyntaxValidator.php

```diff
@@ -27,6 +27,10 @@ class ExpressionLanguageSyntaxValidator extends ConstraintValidator
 
     public function __construct(ExpressionLanguage $expressionLanguage = null)
     {
+        if (null === $expressionLanguage && !class_exists(ExpressionLanguage::class)) {
```

Comment:
I would simplify to:

```suggestion
        if (!class_exists(ExpressionLanguage::class)) {
```

Reply:
I did this to be coherent with : https://github.com/symfony/symfony/blob/9b6dba14b73a5fefda2a9ab1335aef6fb38a00ed/src/Symfony/Component/Validator/Constraints/NotCompromisedPasswordValidator.php#L41